toqito/matrix_ops/tests/test_commutant.py

```diff
@@ -0,0 +1,38 @@
+"""Test the commutant function."""
+import numpy as np
+import pytest
+from toqito.matrix_ops.commutant import commutant
+
+@pytest.mark.parametrize(
+    "matrices, expected_size",
+    [
+        ([np.array([[1, 0], [0, -1]])], 2),  # Pauli-Z: Expect diagonal matrices
+        ([np.array([[0, 1], [1, 0]])], 2),   # Pauli-X: Expect identity and X
+        ([np.array([[0, -1j], [1j, 0]])], 2),  # Pauli-Y: Expect I and Y
+        ([np.eye(2)], 4),  # Identity matrix: Should commute with everything
```

Comment:
[nit]: Could you place the comments directly above the lines to which they refer (instead of to the right?). This is a nit-picky convention, but it aligns with how things are done elsewhere. Don't forget to do so for the other parameterized tests as well.